--- src/plugins/select/Makefile.in	2013-01-17 16:50:36.000000000 +0200
+++ src/plugins/select/Makefile.in	2013-01-17 16:50:45.000000000 +0200
@@ -120,7 +120,7 @@
 	distdir
 ETAGS = etags
 CTAGS = ctags
-DIST_SUBDIRS = bluegene cons_res cray linear serial
+DIST_SUBDIRS = bluegene cons_res cray linear serial lpconsres
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
 am__relativize = \
   dir0=`pwd`; \
@@ -356,7 +356,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 @WITH_CXX_TRUE@BLUEGENE = bluegene
-SUBDIRS = $(BLUEGENE) cons_res cray linear serial
+SUBDIRS = $(BLUEGENE) cons_res cray linear serial lpconsres
 all: all-recursive
 
 .SUFFIXES:
